### ISSUE TITLE
Device wrongly recognized as an Android device

### ISSUE THREAD

**Issue by awsms** [M0Rf30/android-udev-rules#234]
After installing android-tools on my machine, I couldn't play games with my Xbox One controller. Same after rebooting.
Then I did a ls on on /dev/bus/usb/001/id_of_the_controller and noticed it was owned by adbusers. So I digged more into my udev rules and realized it was detected as an android device.
I purged the android-tools package (pacman --remove --recursive android-tools) and it works again now.

output of `udevadm info` when android-udev was still installed
```
P: /devices/pci0000:00/0000:00:08.1/0000:06:00.3/usb1/1-2/1-2.2
M: 1-2.2
R: 2
U: usb
T: usb_device
D: c 189:11
N: bus/usb/001/012
L: 0
S: android
S: android2
V: usb
E: DEVPATH=/devices/pci0000:00/0000:00:08.1/0000:06:00.3/usb1/1-2/1-2.2
E: DEVNAME=/dev/bus/usb/001/012
E: DEVTYPE=usb_device
E: DRIVER=usb
E: PRODUCT=45e/2d1/203
E: TYPE=255/71/208
E: BUSNUM=001
E: DEVNUM=012
E: MAJOR=189
E: MINOR=11
E: SUBSYSTEM=usb
E: USEC_INITIALIZED=18920950679
E: ID_VENDOR=Microsoft
E: ID_VENDOR_ENC=Microsoft
E: ID_VENDOR_ID=045e
E: ID_MODEL=Controller
E: ID_MODEL_ENC=Controller
E: ID_MODEL_ID=02d1
E: ID_REVISION=0203
E: ID_SERIAL=Microsoft_Controller_7EED89303CD2
E: ID_SERIAL_SHORT=7EED89303CD2
E: ID_BUS=usb
E: ID_USB_INTERFACES=:ff47d0:
E: ID_VENDOR_FROM_DATABASE=Microsoft Corp.
E: ID_MODEL_FROM_DATABASE=Xbox One Controller
E: adb_user=yes
E: ID_PATH=pci-0000:06:00.3-usb-0:2.2
E: ID_PATH_TAG=pci-0000_06_00_3-usb-0_2_2
E: ID_FOR_SEAT=usb-pci-0000_06_00_3-usb-0_2_2
E: DEVLINKS=/dev/android /dev/android2
E: TAGS=:seat:uaccess:
E: CURRENT_TAGS=:seat:uaccess:
```

after I uninstalled it
```
P: /devices/pci0000:00/0000:00:08.1/0000:06:00.3/usb1/1-2/1-2.2
M: 1-2.2
R: 2
U: usb
T: usb_device
D: c 189:13
N: bus/usb/001/014
L: 0
V: usb
E: DEVPATH=/devices/pci0000:00/0000:00:08.1/0000:06:00.3/usb1/1-2/1-2.2
E: DEVNAME=/dev/bus/usb/001/014
E: DEVTYPE=usb_device
E: DRIVER=usb
E: PRODUCT=45e/2d1/203
E: TYPE=255/71/208
E: BUSNUM=001
E: DEVNUM=014
E: MAJOR=189
E: MINOR=13
E: SUBSYSTEM=usb
E: USEC_INITIALIZED=19735767534
E: ID_VENDOR=Microsoft
E: ID_VENDOR_ENC=Microsoft
E: ID_VENDOR_ID=045e
E: ID_MODEL=Controller
E: ID_MODEL_ENC=Controller
E: ID_MODEL_ID=02d1
E: ID_REVISION=0203
E: ID_SERIAL=Microsoft_Controller_7EED89303CD2
E: ID_SERIAL_SHORT=7EED89303CD2
E: ID_BUS=usb
E: ID_USB_INTERFACES=:ff47d0:
E: ID_VENDOR_FROM_DATABASE=Microsoft Corp.
E: ID_MODEL_FROM_DATABASE=Xbox One Controller
E: ID_PATH=pci-0000:06:00.3-usb-0:2.2
E: ID_PATH_TAG=pci-0000_06_00_3-usb-0_2_2
```

**Comment by M0Rf30** [comment]
Hi @awsms and thanks for your feedback. Can you confirm that everything works with latest commits?

**Comment by awsms** [comment]
Hello @M0Rf30. This seems all good now, thank you!
